deps/wazuh_testing/wazuh_testing/logcollector.py

```diff
@@ -521,8 +562,19 @@ def delete_file_structure(get_files_list):
     Args:
         get_files_list(dict):  Files to delete.
     """
-    for file in get_files_list:
-        shutil.rmtree(file['folder_path'], ignore_errors=True)
+
+    def remove_readonly(func, path, _):
+        "Clear the readonly bit and reattempt the removal"
```

Comment:
```suggestion
    def remove_read_only(func, path, _):
        """Clear the read_only bit and reattempt the removal
        
        Args:
        ...
        """
```

Reply:
Done in acb2e52280b139c60e2178171fd356f5100ba6c1